src/test/porter/README.md

```diff
@@ -0,0 +1,68 @@
+# porter chart
+
+---
+
+## Introduction
+
+Porter is an open source load balancer designed for bare metal Kubernetes clusters. It's implemented by physical switch, and uses BGP and ECMP to achieve the best performance and high availability.
+
+---
+
+## Pre-requisites
+
+- kubernetes version:1.17.3
```

Comment:
Does this chart only support k8s v1.17.3? Or is this mean the version you just tested?

Reply:
you just tested 1.17.3